next/src/field/object.ts

```diff
@@ -28,7 +28,7 @@ export function buildFieldObject(schema: JsfObjectSchema, name: string, required
     type: schema['x-jsf-presentation']?.inputType || 'fieldset',
     inputType: schema['x-jsf-presentation']?.inputType || 'fieldset',
     jsonType: 'object',
-    name: schema.title || name,
```

Comment:
This just turned out to be wrong. `title` is the human readable name and `name` is the field name that will be used as the html input name.